list_updatable_packages

```diff
@@ -103,15 +103,15 @@ def find_gemfile_lock_specs(ci_job: str, subdir: str) -> Generator[Tuple[Spec, s
         if path.is_file():
             spec = Spec(path)
             if spec.is_updateable:
-                yield spec, gem['version']
+                yield spec, gem['version'], project
 
 
 def find_packaged_specs() -> Generator[Tuple[Spec, None], None, None]:
```

Comment:
```suggestion
def find_packaged_specs() -> Generator[Tuple[Spec, None, None], None, None]:
```

Reply:
Curious, I am surprise issues like this did not throw errors.

Reply:
Type definitions aren't enforced at runtime, but can be used by external tools (and introspected at runtime). So your IDE can provide better auto completion suggestions. https://pydantic.dev/ is a library that does use the type definitions at runtime to (de)serialize data, but also validate it.